lightly/embedding/embedding.py

```diff
@@ -123,16 +133,26 @@ def embed(self,
 
                 process_time = time.time()
 
-                efficiency = \
-                    (process_time - prepare_time) / (process_time - start_time)
-                pbar.set_description(
-                    "Compute efficiency: {:.2f}".format(efficiency))
+                efficiency = (process_time - prepare_time) / (process_time - start_time)
```

Comment:
While you're at it, I have the feeling that the efficiency calculation is buggy... Do you think it makes sense like this?

Reply:
I think that the calculation was correct. However, I refactored it a little to make it better understandable what is done.